tests/compat.sh

```diff
@@ -13,6 +13,10 @@
 # Check each common ciphersuite, with each version, both ways (client/server),
 # with and without client authentication.
 
+# Limit log output to 10 GB, in case of failures with this script where it may
```

Comment:
The comment says 10GB, but the code says 10*2^20 units of 512B (`ulimit -f` is expressed in 512B units in sh — not in bash, but we're running sh here, and even if sh is implemented by bash, bash does enter a compatible mode when it's invoked as sh).